ggml-cuda.cu

```diff
@@ -7403,7 +7410,7 @@ static void ggml_cuda_mul_mat_mat_batched_cublas(const ggml_tensor * src0, const
 
 static void ggml_cuda_mul_mat(const ggml_tensor * src0, const ggml_tensor * src1, ggml_tensor * dst) {
     const bool all_on_device =
-        (src0->backend == GGML_BACKEND_GPU) &&
+        (src0->backend == GGML_BACKEND_GPU || src0->backend == GGML_BACKEND_GPU_SPLIT) &&
```

Comment:
The `ggml_cuda_mul_mat_mat_batched_cublas` branch does not currently support split tensors.
Would this break F16 models for multi-gpu?

Reply:
yes, it is broken